src/Workspaces/Remote/Core/RemoteHostAssetSerialization.cs

```diff
@@ -137,6 +137,29 @@ public static async ValueTask<ImmutableArray<(Checksum, object)>> ReadDataAsync(
                 // reader and/or writer while they are still in use.
                 await copyTask.ConfigureAwait(false);
             }
+
+            // Local functions
+            static bool IsExpectedEndOfStreamException(EndOfStreamException e, Exception? copyException, CancellationToken cancellationToken)
```

Comment:
so.  oddly enough, 'e' is never even used.  is it passed it for clarity purposes?  is it ok to not use?  should we doc that, while strange, it isn't needed, and the `return false;` at the end will ensure it propagates if it's the exception of real interest?

Reply:
Updated to remove the unused variable

Reply:
wfm.